esx_service/utils/auth.py

```diff
@@ -91,34 +91,34 @@ def get_default_tenant():
         return None, None, None
 
 
-def get_default_privileges():
-    """ Get DEFAULT privileges by querying the auth DB.
-        DEFAULT privilege will match any (tenant_uuid, datastore) pair which
-        does not have an entry in privileges table explicitly
+def get_all_ds_privileges(tenant_uuid):
+    """ Get privileges on _ALL_DS by querying the auth DB.
+        For the tenant with given tenant_uuid, this privilege
+        will match any datastore which does not have an entry
+        in privileges table explicitly
```

Comment:
@govint Yes. We should document it. In typical use case, "ALL_DS" is created by our code only for "DEFAULT" tenant after install. We don't recommend user to add a privilege to "ALL_DS" for named tenant.